cardano-cli/src/Cardano/CLI/Byron/Commands.hs

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE GeneralisedNewtypeDeriving #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
```

Comment:
Is this commit still needed?

Reply:
I've removed the pragma changes to see what happens.

Reply:
Looks to be working.